android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
         versionName "1.12"
         vectorDrawables.useSupportLibrary = true
         missingDimensionStrategy 'react-native-camera', 'general'
+        //multiDexEnabled true
```

Comment:
Is this going to used commonly? If it's not going to be used ever it should just be removed instead of commented